### PR TITLE
fix radio and checkbox tests

### PR DESCRIPTION
**Pull request by umahaea**
Fix tests for radio and checkboxes as per the new config.
Clean buildJSONConfig() method.

### CODE REVIEW

File: src/test/java/elementsSDKTests/functionalTests/InputsTest.java [hunk 1]
```diff
@@ -122,7 +127,7 @@ private void singleLineTextInputWiWoValuesTest(String type, By element, String[]
     @Test(testName = "Fancy - Verify Single Line Text Input", dataProvider = "Fancy - Single Line Text Input Test Data", groups = {"desktop-ci", "desktop-regression"})
     private void fancySingleLineTextInputTest(String cssProperty, String[] expectedCSSValue) throws Exception {
         String[] detailsPropertiesList = new String[]{"elementId", "text-input-target", "componentName", "TextInput"};
-        String[] propsPropertiesList = new String[]{"id", "sl-text-input", "labelText", "Last Name", "inputType", "default", "fancy", "true", "placeholder", "Last Name", "changeHandler", "function () {}", "infoMessage", "This is an info message", "errorMessage", "This is an error message"};
+        String[] propsPropertiesList = new String[]{"id", "sl-text-input", "labelText", "Last Name", "inputType", "default", "fancy", "true", "placeholder", "Last Name", "changeHandler", "function () {}", "infoMessage", "This is an info message", "errorMessage", "This is an error message'"};
```
Comment (udhadpa) [inline src/test/java/elementsSDKTests/functionalTests/InputsTest.java]: observing you have put single quotes at the end of this string, and not in the beginning - Password is visible'
Is this required for correctly parsing the config?

Comment (umahaea) [reply src/test/java/elementsSDKTests/functionalTests/InputsTest.java]: yes Payal, for some reason the buildJSONConfig() doesn't include a " ' " for the last item, I tried my best.

Comment (udhadpa) [reply src/test/java/elementsSDKTests/functionalTests/InputsTest.java]: cool 👍

**Comment by udhadpa** [comment]
Looks good. 👍

**Comment by umahaea** [comment]
many approved. merging it.

**Pull request merged**